Safewaan/HonkedIn
# Create starter pages

## Trello ticket link

N/A



## Implementation description
* Two starter pages, Sign In, and Landing, have been created.
* The Sign In page has a button that redirects to the Landing page.
* The Landing page contains an appbar with buttons (which will be used in the future to redirect to other pages).



## Steps to test
1. Deploy the application.
2. Verify that the application starts at the Sign In page.
3. Verify that the "Sign In" button redirects to the landing page.
4. Verify that the Landing page has the '/Landing' endpoint.



## What should reviewers focus on?
* Note: the buttons on the Landing page currently have no functionality, but whoever will be working on that feature (ex: My Profile) will have to create the correct endpoints and ensure the button redirects to the respective pages.
* More buttons for the pages can be added later, or removed as needed.
